kafka/kafka.go

```diff
@@ -125,6 +125,15 @@
 // These errors are normally just informational since the
 // client will try its best to automatically recover (eventually).
 //
+// * `OAuthBearerTokenRefresh` - time to retrieve a new SASL/OAUTHBEARER token.
```

Comment:
"time to" might be interpreted as this is an actual time(stamp), might be better with "request to" or similar.

Reply:
👍